### PR TITLE
tikv-ctl: support get range properties under remote mode

### LINKED ISSUE #14620: tikv-ctl range-properties supports remote mode
### Is your feature request related to a problem? Please describe:

Currently, tikv-ctl [range-properties only supports local mode](https://github.com/tikv/tikv/blob/4b5846f85b7f6a58c3d44f764d4fe722fccaa64b/cmd/tikv-ctl/src/executor.rs#L844). In local mode, users need to stop the running TiKV instance, so it's inconvenient for users to use in online scenarios.
### Describe the feature you'd like:

tikv-ctl range-properties supports remote mode.

### PR DESCRIPTION
**Pull request by jinshayumi**
### What is changed and how it works?

Issue Number: Close #14620 

What's Changed: support get range properties under remote mode

### Related changes

- PR to update `pingcap/kvproto`:

### Check List 

Tests 

- No code

### Release note 

```release-note
Make get_range_properties command support remote mode.
```

### CODE REVIEW

**Review by pingyu — APPROVED** [review]
LGTM~

File: src/server/service/debug.rs [hunk 1]
```diff
@@ -433,6 +433,35 @@ where
...
+        let f =
+            self.pool
+                .spawn(async move {
+                    debugger.get_range_properties(req.get_start_key(), req.get_end_key())
+                })
+                .map(|res| res.unwrap())
+                .map_ok(|props| {
+                    let mut resp = GetRangePropertiesResponse::default();
+                    for (key, value) in props {
+                        let mut prop = get_range_properties_response::RangeProperty::default();
```
Comment (bufferflies) [inline src/server/service/debug.rs:454]: maybe you should follow the code line get_region_properties

**Review by bufferflies — APPROVED** [review]

**Pull request merged**